packages/fluentui/react-bindings/src/styles/resolveStyles.ts

```diff
@@ -127,7 +130,7 @@ const resolveStyles = (
   const propsCacheKey = cacheEnabled ? JSON.stringify(stylesProps) : '';
   const variablesCacheKey = cacheEnabled && performance.enableBooleanVariablesCaching ? JSON.stringify(variables) : '';
   const componentCacheKey = cacheEnabled
-    ? `${displayName}:${propsCacheKey}:${variablesCacheKey}:${styleParam.rtl}${styleParam.disableAnimations}`
+    ? `${displayNames.join(':')}:${propsCacheKey}:${variablesCacheKey}:${styleParam.rtl}${styleParam.disableAnimations}`
```

Comment:
nit: maybe extract this in a utility function, or at least store the separator in a const.

Reply:
I am not sure that it will make code there more readable, so will skip this change for now. Obviously we will back there and can reconsider it 👍